README.md

```
@@ -14,35 +18,31 @@
+   * Once you are finished, verify that the check [Validate models.csv](../../actions/workflows/check-models.yml) was successful!
 1. [OPTIONAL] Adjust the OSP environment and tools by updating [`tools.csv`](tools.csv)<br>(see the [Tools](#tools) section below for details)
    * In case of modifications: verify that the check [Validate tools.csv](../../actions/workflows/check-tools.yml) was successful!
 1. Go to the GitHub Action: [Create evaluation reports and projects](../../actions/workflows/create-evaluation_reports.yml)<br><br>
    * Click the __Run workflow__ button 
-   * Select the branch defined in the first step (for instance, `my-reports`)
    * [OPTIONAL] Adjust the commit message that will appear later in the created pull request<br>(see the [What to do when reports are created](#next-step) section below).
    * Click the green __Run workflow__ button
-     
-        <img width="800" alt="Run workflow dialog showing branch selection and commit message options" src="https://github.com/user-attachments/assets/d787ddc6-db2a-47c0-ab98-82bcb0051bb4" />
+  
+        <img width="700" src="https://github.com/user-attachments/assets/b2c02567-4448-4196-b0bb-1367ab46ecce" />
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)